src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -260,6 +260,7 @@ export const searchEmptyState = {
         const searchBar = canvas.getByRole('searchbox', { name: 'Search files and folders' });
         await userEvent.type(searchBar, 'foo');
 
+        expect(canvas.getByText('test')).toBeInTheDocument();
```

Comment:
test

Reply:
if you look inside of the VRT the word `test` is the flaky part because the name of the folder is called test, adding the assertion needs to give it enough time